### ISSUE TITLE
Tab/keyboard order for FL "search this area" button 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#19363]
## Issue Description
As reported by Josh, "search this area" button is not reachable in the tab order (on any browsers).  It is only accessible when focus is moved to it on the first time the user fires a zoom in or out.

Per @joshkimux the search button focus should either
1- be persistent
2- or added to the tab order when in flow

## Task
- Remove the canvas from the tab order (to which we cannot apply styles). After the container, the zoom controls should get focus

## Acceptance Criteria
**On any browser**
- [ ] Anytime a Veteran fires a zoom in/out or pans the map, the "search this area" button receives focus and is reachable by keyboard in a logical order. 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by joshkimux** [comment]
@mmiddaugh this should be considered from a design perspective a little bit more! It may be frustrating for keyboard users who are wanting to zoom in multiple times to have the focus constantly shifted to the search this area button. An alternative may be to have the search this area button be in the DOM but initially hidden-- then make it appear and use aria-live polite to provide instructions to screen reader users that it is now reachable by tabbing.

**Comment by mmiddaugh** [comment]
Addressed during #21256
